From e14face2f323388df053953ee7c20fb7171a8a60 Mon Sep 17 00:00:00 2001
From: Stuart Yoder <stuart.yoder@nxp.com>
Date: Fri, 15 Jan 2016 15:09:36 -0600
Subject: [PATCH 006/245] configs: add config fragment for freescale

enables the following

 -ramdisk support
 -automatically append version info to version str
 -48-bit virtual addresses (allows > 2GB of memory)
 -modversions,module_force_load
 -slab allocator
 -printk timestamp
 -bus driver
 -smmu

Signed-off-by: Stuart Yoder <stuart.yoder@nxp.com>
---
 arch/arm64/configs/freescale.config | 15 +++++++++++++++
 1 file changed, 15 insertions(+)
 create mode 100644 arch/arm64/configs/freescale.config

diff --git a/arch/arm64/configs/freescale.config b/arch/arm64/configs/freescale.config
new file mode 100644
index 0000000..8e8588a
--- /dev/null
+++ b/arch/arm64/configs/freescale.config
@@ -0,0 +1,15 @@
+# general options
+CONFIG_LOCALVERSION_AUTO=y
+CONFIG_SLAB=y
+CONFIG_MODULE_FORCE_LOAD=y
+CONFIG_MODVERSIONS=y
+CONFIG_ARM64_VA_BITS_48=y
+CONFIG_BLK_DEV_RAM=y
+CONFIG_BLK_DEV_RAM_SIZE=262144
+CONFIG_PRINTK_TIME=y
+# iommu
+CONFIG_IOMMU_SUPPORT=y
+CONFIG_ARM_SMMU=y
+# dpaa2
+CONFIG_STAGING=y
+CONFIG_FSL_MC_BUS=y
-- 
2.1.0.27.g96db324

